src/structures/Node.ts

```diff
@@ -101,7 +101,7 @@ export class Node {
         this.password = options.password;
         this.retryAmount = options.retryAmount;
         this.retryDelay = options.retryDelay;
-        this.baseURL = `http${options.secure ? "s" : ""}://${options.host}${options.port ? `:${options.port}` : ""}/`;
+        this.baseURL = `http${options.secure ? "s" : ""}://${options.host}${options.port ? `:${options.port}` : ""}`;
```

Comment:
If u will remove slash that means you will need to add / in search and subscription url

Reply:
No need in `makeRequest`, i have already added it.

Reply:
I see